Dockerfile_archive

```diff
@@ -37,3 +37,5 @@ RUN pip3 install --break-system-packages --upgrade "/code[gcp,archive]" \
 EXPOSE 9914
 
 ENTRYPOINT ["janitor-archive", "--port=9914", "--listen-address=0.0.0.0"]
+
+CMD ["--help"]
```

Comment:
Why specify --help here? You can specify --help on the command-line, and for some of these docker images there is decent default behaviour if you don't specify any arguments. It's also not common practice with docker images to default to --help.

Reply:
Looking at docker: https://docs.docker.com/build/building/best-practices/

```
The following is an example of an image for the command line tool s3cmd:

[ENTRYPOINT](https://docs.docker.com/reference/dockerfile/#entrypoint) ["s3cmd"]
[CMD](https://docs.docker.com/reference/dockerfile/#cmd) ["--help"]
```

Reply:
In that case, let's improve the code to print an error message and print a usage message if a parameter is missing / file is missing.

Reply:
I will remove the `--help` CMD part.

is it also worth dropping the default value of `--config`'s `janitor.conf` ?

Reply:
Dropped